### ISSUE TITLE
An error occurred while handling a request: Cannot modify header after it's been flushed.

### ISSUE THREAD

**Issue by kambrium** [andrewlalis/handy-httpd#11]
First of all: Thanks for this great library! :clap: 

I use handy-httpd as a static file server in my repo [dwttool](https://github.com/kambrium/dwttool/blob/master/source/app.d#L268). On a first glance, everything works well. The responses look good. However, there are the following two errors in the log whenever I request a file.

```
handy_httpd.components.handler ERROR 2023-03-12T14:29:06.793 An error occurred while handling a request: Cannot modify header after it's been flushed.
handy_httpd.components.handler ERROR 2023-03-12T14:29:06.793 The response has already been sent; cannot send 500 error.
```

What did I do wrong?

I use 5.6.0.

**Comment by andrewlalis** [comment]
Sorry for the delay; was on vacation last week. I'll look into this, because it looks like an issue with the `FileResolvingHandler`'s implementation which wasn't tested because I wasn't able to set up integration tests for the pre-made handlers yet...

**Comment by andrewlalis** [comment]
Okay, it should be fixed with version `5.6.1`.

**Comment by andrewlalis** [comment]
Okay, as you might see in the pipelines, there are some issues with the latest LDC version and the Httparsed library that Handy-httpd uses, but if you're using dmd it should still be fine. I'll open a separate issue to track those changes.

**Comment by kambrium** [comment]
It works. Thanks!

**Comment by andrewlalis** [comment]
@kambrium just letting you know in case you use LDC as your compiler, there is currently an issue where some built-in functions are missing in LDC v1.32.0, and that affects Handy-httpd, but it'll be fixed in LDC v1.32.1 soon.
